src/translators/sql_translator.py

```diff
@@ -946,8 +969,20 @@ def _format_response(self, resultset, col_names, table_name, last_n):
         """
         stmt = "select entity_attrs from {} " \
                "where table_name = ?".format(METADATA_TABLE_NAME)
-        self.cursor.execute(stmt, [table_name])
-        res = self.cursor.fetchall()
+        try:
+            # TODO we tested using cache here, but with current "delete" approach this causes issues
+            # scenario triggering the issue is: a entity is create, delete is used to delete all values
+            # what happens is that the table is empty, but metadata are still there, so caching the query with
+            # res = self._execute_query_via_cache(table_name, "metadata", stmt, [table_name], self.default_ttl)
+            # actually create an entry in the cache table_name, "metadata"
+            # in a following query call (below ttl) the same cache can be called despite there is no data.
+            # a possible solution is to create a cache based on query parameters that would cache all the results
+            self.cursor.execute(stmt, [table_name])
+            res = self.cursor.fetchall()
```

Comment:
we tested using cache here, but with current "delete" approach this causes issues scenario triggering the issue is: a entity is create, delete is used to delete all values what happens is that the table is empty, but metadata are still there, so caching the query with
```
res = self._execute_query_via_cache(table_name, "metadata", stmt, [table_name], self.default_ttl)
```
actually create an entry in the cache table_name, "metadata", in a following query call (below ttl) the same cache can be called despite there is no data. a possible solution is to create a cache based on query parameters that would cache all the results